source/reference/limitations.txt

```diff
@@ -54,6 +54,11 @@ General Limitations
 - `Queryable Encryption
   <https://www.mongodb.com/docs/v6.0/core/queryable-encryption/>`__ is
   not supported.
+- If you replace the ``mongosync`` binary during an upgrade or a
```

Comment:
This isn't quite right. It _will try_ to resume from where it left off. But we don't guarantee that will work. That's because we store in the destination cluster about the state of the sync process. We don't guarantee that this data will be compatible with other mongosync versions. So it's possible that it will either A) just work fine and resume where you left off; B) crash; C) do something else surprising.

Right now we want to tell people to drop the existing sync data (or just wipe the destination cluster entirely) as part of the upgrade/downgrade process. I see that we _do_ say this in the upgrade/downgrade section, but should we spell that out here as well?